requirements.in

```diff
@@ -0,0 +1,23 @@
+cirq==0.9.1
+seaborn
+sphinx
+ipython
+pytest
+numpy
+
+# optimize
+scikit-learn
+
+# qaoa
+networkx
+pytket-cirq==0.3
+pytket==0.5.2
+
+# quantum chess, only needed for tests
+scipy
```

Comment:
you can put this up in the top group of "general numeric python requirements". Along with numpy (and pandas and matplotlib) it's (or at least it was) assumed to get roped in from cirq